tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# tasks file for ansible-playbook-telegraf
+#
+
+- name: fetch telegraf installation package
+  get_url: url=http://get.influxdb.org/telegraf/telegraf_{{ telegraf_version }}_amd64.deb dest=/tmp/telegraf_{{ telegraf_version }}_amd64.deb mode=0440
+
+- name: install telegraf package
+  command: dpkg --skip-same-version -i /tmp/telegraf_{{ telegraf_version }}_amd64.deb
+  register: dpkg_result
+  changed_when: "dpkg_result.stdout.startswith('Selecting')"
+
+- name: start telegraf service and ensure that it is enabled
+  service: name=telegraf enabled=yes state=started
```

Comment:
We did start out with this config the other way around, but it complains that the destination directory didn't exist - that directory is created by telegraf when it first starts up. We could have it before, but we'd have to create the directory with ansible if we want to swap the order over.